apps/api/src/modules/payments/payments.service.ts

```diff
@@ -736,7 +737,54 @@ export default class PaymentsService {
         externalId: sourcePayment.externalId,
         status: sourcePayment.status,
       })
+      // Send Awaiting payment notification to customer service
+      if (Configuration.customerServiceEmail) {
+        const packTemplate = await this.packs.getPackById(payment.packId)
+        await this.notifications.createNotification(
+          {
+            type: NotificationType.CSAwaitingWirePayment,
+            userAccountId: payment.payerId,
+            variables: {
+              packTitle: packTemplate.title,
+              amount: sourcePayment.amount,
+            },
+          },
+          trx
+        )
+      }
+    }
+
+    // Send email notification to  Customer service
+    if (Configuration.customerServiceEmail) {
```

Comment:
This is the same as the check on line  741, I think the branching could be simplified.